### PR TITLE
dev-ruby/rubygems: add GPL-2 to the LICENSE to cover init script ... and switch to --user

### PR DESCRIPTION
**Pull request by henning-schild**
Using GPL-2 for the init script would add that licence to this ebuild,
while the original software is not GPL-2. Simply remove licence
information to get rid of that problem.
Also change --chuid to --user to get rid of warning when starting, and
bump ebuild to -r1 to force remerge.

Bug: https://bugs.gentoo.org/425936
Signed-off-by: Henning Schild <henning@hennsch.de>

### CODE REVIEW

File: dev-ruby/rubygems/files/init.d-gem_server2 [hunk 1]
```diff
@@ -1,6 +1,4 @@
 #!/sbin/openrc-run
```
Comment (Zlogene) [inline dev-ruby/rubygems/files/init.d-gem_server2]: No, it does not work this way. See https://bugs.gentoo.org/425702

**Comment by Zlogene** [comment]
And again, it does not work this way, an init script as a matter of property belongs the Gentoo Found action, and, therefore can only be destributed under GPLv2

**Comment by henning-schild** [comment]
@Zlogene the bug you mentioned says

> The general preference should be to relicense the script to either whatever is used in LICENSE.
> Adding GPL-2 to the LICENSE variable is also acceptable. A third option is to relicense the script
> BSD-2 to be consistent with OpenRC's init scripts and then add BSD-2 to LICENSE.

So the path i chose is the preferred one, according to that. Please just tell me what to implement and i will go for that.

**Comment by Zlogene** [comment]
well, I would just include GPLv2 to the license line of the ebuild, otherwise it creates a mess

**Comment by graaff** [comment]
Merged.

**Pull request closed without merging**